model-optimizer/extensions/front/tf/InterpolateTransposes.py

```diff
@@ -16,49 +16,56 @@
 import numpy as np
 
 from mo.front.common.partial_infer.utils import int64_array
-from mo.front.tf.replacement import FrontReplacementFromConfigFileGeneral
-from mo.graph.graph import Graph, Node
-from mo.middle.pattern_match import find_pattern_matches, inverse_dict
+from mo.front.tf.replacement import FrontReplacementSubgraph
+from mo.graph.graph import Graph
 
 
-class InterpolateTranspose(FrontReplacementFromConfigFileGeneral):
+class InterpolateTranspose(FrontReplacementSubgraph):
     """
     Delete useless transposes around ResizeNearestNeighbor op. In TF this op is working in NHWC layout,
     Resample in OpenVINO working in NCHW layout. If all graph has NCHW layout we should delete transposes around
     Resample: (NCHW->NHWC) -> Resample -> (NHWC -> NCHW) to run this op in NCHW without changes of layout.
     """
     enabled = True
-    replacement_id = 'InterpolateTranspose'
     graph_condition = [lambda graph: graph.graph['layout'] == 'NCHW']
 
-    pattern_nodes = [
-        ('interpolate', {'kind': 'op', 'op': 'Interpolate'}),
-        ('transpose_1', {'kind': 'op', 'op': 'Transpose'}),
-        ('transpose_1_order', {'kind': 'op', 'op': 'Const',
-                               'value': lambda x: x is not None and np.array_equal(x, int64_array([0, 2, 3, 1]))}),
-        ('transpose_2', {'kind': 'op', 'op': 'Transpose'}),
-        ('transpose_2_order', {'kind': 'op', 'op': 'Const',
-                               'value': lambda x: x is not None and np.array_equal(x, int64_array([0, 3, 1, 2]))}),
-    ]
-    pattern_edges = [
-        ('transpose_1', 'interpolate', {'in': 0, 'out': 0}),
-        ('transpose_1_order', 'transpose_1', {'in': 1, 'out': 0}),
-        ('interpolate', 'transpose_2', {'in': 0, 'out': 0}),
-        ('transpose_2_order', 'transpose_2', {'in': 1, 'out': 0}),
-    ]
+    def pattern(self):
```

Comment:
This transformation should be invoked after `TransposeOrderNormalizer`

Reply:
Fixed.